docs/diagnostics/index.rst

```diff
@@ -8,10 +8,17 @@ Plasma diagnostics (`plasmapy.diagnostics`)
 
 .. automodapi:: plasmapy.diagnostics.langmuir
    :no-heading:
+   
+.. currentmodule:: plasmapy.diagnostics.thomson
+
+.. automodapi:: plasmapy.diagnostics.thomson
+   :no-heading:   
```

Comment:
Oh right! I missed it, but this is not going to be necessary; it'll be simpler for me to send you a PR to this PR :laughing: Expect it soon!

Reply:
Done!